test/interval_tests/numeric.jl

```diff
@@ -35,17 +35,17 @@ setprecision(Interval, Float64)
     @test Interval(-1.0, 1.0)/Interval(0.0,1.0) == entireinterval(c)
     @test Interval(-1.0, 1.0)/Interval(-1.0,1.0) == entireinterval(c)
 
-    @test extended_div(Interval(-30.0,-15.0), Interval(-5.0,-3.0)) == Interval(3.0, 10.0)
-    @test extended_div(@interval(-30,-15) , @interval(-5,-3)) == Interval(3.0, 10.0)
-    @test extended_div(1.0..2.0, 0.1..1.0) == Interval(1, 20)
-    @test extended_div(a, c) == Interval(2.4999999999999998e-02, 4.4e+00)
-    @test extended_div(c, Interval(4.0)) == Interval(6.25e-02, 1e+00)
-    @test extended_div(c, zero(c)) == emptyinterval(c)
-    @test extended_div(Interval( 0.0, 1.0), Interval(0.0,1.0)) == Interval(0.0, Inf)
-    @test extended_div(Interval(-1.0, 1.0), Interval(0.0,1.0)) == entireinterval(c)
-    @test extended_div(Interval(-1.0, 1.0), Interval(-1.0,1.0)) == entireinterval(c)
-    @test extended_div(Interval(1.0, 2.0), Interval(-4.0, 4.0)) == [(-∞.. -0.25), (0.25..∞)]
-    @test extended_div(Interval(-1.0, -2.0), Interval(-2.0, 4.0)) == [(-∞.. -0.5), (1.0..∞)]
+    @test extended_div(Interval(-30.0,-15.0), Interval(-5.0,-3.0)) == (Interval(3.0, 10.0), emptyinterval(c))
+    @test extended_div(@interval(-30,-15) , @interval(-5,-3)) == (Interval(3.0, 10.0), emptyinterval(c))
+    @test extended_div(1.0..2.0, 0.1..1.0) == (Interval(1, 20.000000000000004), emptyinterval(c))
+    @test extended_div(a, c) == (Interval(2.4999999999999998e-02, 4.4e+00), emptyinterval(c))
+    @test extended_div(c, Interval(4.0)) == (Interval(6.25e-02, 1e+00), emptyinterval(c))
+    @test extended_div(c, zero(c)) == (emptyinterval(c), emptyinterval(c))
+    @test extended_div(Interval( 0.0, 1.0), Interval(0.0,1.0)) == (Interval(0.0, Inf), emptyinterval(c))
+    @test extended_div(Interval(-1.0, 1.0), Interval(0.0,1.0)) == (entireinterval(c), emptyinterval(c))
+    @test extended_div(Interval(-1.0, 1.0), Interval(-1.0,1.0)) == (entireinterval(c), emptyinterval(c))
+    @test extended_div(Interval(1.0, 2.0), Interval(-4.0, 4.0)) == ((-∞.. -0.25), (0.25..∞))
+    @test extended_div(Interval(-1.0, -2.0), Interval(-2.0, 4.0)) == ((-∞.. -0.5), (1.0..∞))
```

Comment:
You should not use `Interval` to construct intervals, but rather `interval`. 
Here, for example, `Interval(-1.0, -2.0)` gives an *invalid* interval since -1 > -2.

```
julia> interval(-1, -2)
ERROR: ArgumentError: `[-1, -2]` is not a valid interval. Need `a ≤ b` to construct `interval(a, b)`.
```

Reply:
This test seems to be wrong:
```
julia> (-2.. -1) / (-2..0)
[0.5, ∞]

julia> (-2.. -1) / (0..4)
[-∞, -0.25]
```
So maybe your code is wrong?

Reply:
Or maybe it is just coming from using the invalid interval.

Reply:
You should add a test with an interval that contains 0 divided by another interval that contains 0.

Reply:
Sorry, I accidentally put `Interval(-1.0, -2.0)` instead of `Interval(-2.0, -1.0)`. Fixing that should fix this test case. I will also replace `Interval` with `interval`.